code/modules/surgery/organs/internal/cyberimp/augments_arms.dm

```diff
@@ -408,9 +408,10 @@
 		return
 
 	var/mob/living/living_target = target
-
+	var/target_zone = living_target.get_random_valid_zone(source.zone_selected)
+	var/armor_block = living_target.run_armor_check(target_zone, MELEE)
```

Comment:
Could you move these down to `living_target.apply_damage(punch_damage, BRUTE, target_zone, armor_block)` ? Or is there a reason they're so far down?

Reply:
yeah i think i can do that, i don't really know why i nudged those up there